### ISSUE TITLE
Missing audio using a viafree url.

### ISSUE THREAD

**Issue by x-loser-x** [spaam/svtplay-dl#1294]
svtplay-dl versions:

    svtplay-dl 2.8+4.g82265da

Operating system and Python version:

    Ubuntu 18.04.4 LTS x86_64 kernel 5.4.0-54-generic
    python --version: Python 2.7.17
    python3 --version: Python 3.6.9

What is the issue:

There is no issue during the actual download but after muxing and all there is only audio for 6 min 50 sec. then there is no sound at all in the produced mp4. Example url: https://www.viafree.se/program/drama/vara-basta-ar/sasong-25/avsnitt-13967

I'm not providing a debug output since this url now days require a user and password to download...

Still there must be something wrong since there is not all audio merged into the video.

Note: the only switches except -u and -p is -S to get the subtitles for the episode.

**Comment by granbom** [comment]
I downloaded it (HLS, bitrate 4080), and there were no issues. Maybe there was a temporary glitch during your download?

Only other issue I can think about is that you are using an outdated ffmpeg. Check that the ffmpeg version  is at least > 4.
I did notice there was a warning in the audio.ts file

`[aac @ 0x558646c64100] Estimating duration from bitrate, this may be inaccurate`

Older ffmpeg versions might handle it wrong.
